.circleci/config.yml

```diff
@@ -144,14 +144,23 @@ jobs:
       name: go/default
       tag: "1.13"
     steps:
+      - checkout
+      - run:
+          name: "Enforce Go Formatted Code"
+          command: >
```

Comment:
Thanks for this, it will be great to have the check against PRs.

Would it workto use the 'make format', e.g. in general have the same command setup via make that gets performed in CI

Reply:
Good point. Done.